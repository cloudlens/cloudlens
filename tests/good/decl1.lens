process  {
    print(1)
}

lens f(x, log){
    var y = 10; 
    process  {
     	print(x + y)
     }
}

var y = 40;

process  {
    print(y + 2)
}

f(3)

