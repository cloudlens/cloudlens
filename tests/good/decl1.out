1
42
13
1
42
13
1
42
13
1
42
13
